modular_citadel/code/game/machinery/vending.dm

```diff
@@ -26,7 +26,7 @@
 					/obj/item/storage/hypospraykit/toxin = 2,
 					/obj/item/storage/hypospraykit/o2 = 2,
 					/obj/item/storage/hypospraykit/brute = 2,
-					/obj/item/storage/box/vials = 1)
+					/obj/item/reagent_containers/glass/bottle/vial/small = 5)
```

Comment:
This is an alright change, though the chem master will be able to produce vials soon enough